harvard-dcu.csl

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Harvard (Dublin City University)</title>
+    <title-short>Harvard (DCU)</title-short>
+    <id>http://www.zotero.org/styles/harvard-dcu</id>
+    <link rel="self" href="http://www.zotero.org/styles/harvard-dcu"/>
+    <link href="http://www.zotero.org/styles/harvard" rel="template"/>
```

Comment:
@hylandj, we always like to see a "template" link that points to the CSL style your started out with (assuming you based your style on another). We don't have a "harvard.csl" style in our repository, so our tests currently give an error. Can you change this link to one of an existing style?

See also https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link.